src/rockstor/storageadmin/views/rockon_id.py

```diff
@@ -145,13 +145,11 @@ def post(self, request, rid, command):
                         cco.val = cc_map[c]
                         cco.save()
                     for e in env_map.keys():
-                        if (not DContainerEnv.objects.filter(
+                        if (DContainerEnv.objects.filter(
```

Comment:
Finally had my "welcome to Rockons world", it was fun! and I say "No, I don't agree on this"

Let me explain @anatox:
- Rockstor Rockons db entries filled from json files, found it reading/simulating code from rockon.py under storageadmin/views
- On every Rockons "Update" button click we perfor a POST request to rockon.py view to sync db env vars to real json files vars
- [Here](https://github.com/rockstor/rockstor-core/blob/master/src/rockstor/storageadmin/views/rockon.py#L96) we start our post request, getting available rockons (`rockons = self._get_available()`) then updating db entries with `self._create_update_meta(r, rockons[r])`
- `_create_update_meta` then updates env vars too [here](https://github.com/rockstor/rockstor-core/blob/master/src/rockstor/storageadmin/views/rockon.py#L304) and finally `_update_env` has its `get_or_create` over DContainerEnv table

After this long talk, my opinion:
- A: errors on rockon.py when creating db entries
- B: wrong json scrutures
- A+B

Current PR is a solution, but so doing we don't check & fix the real issue: it's like hiding broken floor tiles with a carpet, we don't see them anymore, but they're still there 😉 

Please see this as an invitation to contribute 😃